oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/search/SearchAppLayout.jsx

```diff
@@ -134,19 +135,23 @@ export const SearchAppLayout = ({ config, hasButtonSidebar }) => {
             </Grid.Column>
           )}
           {facetsAvailable && (
-            <Grid.Column floated="left" only="computer" width={11}>
-              <ActiveFilters />
-            </Grid.Column>
+            <ShouldActiveFiltersRender>
+              <Grid.Column floated="left" only="computer" width={11}>
+                <ActiveFilters />
+              </Grid.Column>
+            </ShouldActiveFiltersRender>
           )}
           <Grid.Column textAlign="right" floated="right" {...resultSortLayout}>
             <ResultOptionsWithState />
           </Grid.Column>
         </Grid.Row>
-        <Grid.Row>
-          <Grid.Column floated="left">
-            <ClearFiltersButton />
-          </Grid.Column>
-        </Grid.Row>
+        <ShouldActiveFiltersRender>
+          <Grid.Row>
+            <Grid.Column floated="left">
+              <ClearFiltersButton />
```

Comment:
ClearFiltersButton also renders as <ShouldActiveFiltersRender>, isn't that duplicit?

Reply:
I get your point, in that case, I would remove it from the button itself and leave it to the creator of specific SearchAppLayout component. Layout may differ, and it could also be wrapped together with ActiveFilters under single ShouldActiveFiltersRender element